### DISCUSSION TITLE
[Feature Request] Please add RSS or Atom feed.

### DISCUSSION THREAD

**Discussion by Artnel** [signalstickers/signalstickers#296] [Ideas / feature request]
RSS or Atom feed will allow for getting know about new stickers without opening website and unnecessary usage of data transfer.

**Comment by darkobits** [comment]
This is something we can definitely consider. Can you elaborate on your specific use-case a little more, as this may help inform any future implementation? Thanks!

**Comment by Artnel** [comment]
I use RSS/Atom reader that aggregates rss feeds that I have subscribed. Reader notifies me about new articles/forum_posts/etc on sites and provides headers of them (in case of signalstickers something like: New <original | animated | etc> <sticker pack name> for Signal and maybe exemplary sticker like on main site) . If I'm waiting for new articles or find description interesting I'm checking the article. 
It would be nice to have RSS feed on signalstickers, this way I would be informed about new stickers without the necessity to visit website.
For example this site with stickers for Telegram has RSS feed that I would use if i were using Telegram:
https://telegramchannels.me/stickers

**Comment by quintrino** [comment]
Came here to post this exact feature. I think in terms of functionality, it would be great if each entry in the RSS included the relevant tags (animated/NSFW/etc)

**Comment by romainricard** [comment]
Hi @Artnel @quintrino , thanks again for the suggestion! I've (finally) developed a RSS/Atom feed for Signalstickers. I don't use RSS myself so I'm not very familiar with its customs. Let me know if it works for you!

+ Atom:  https://api.signalstickers.com/feed/atom/
+ RSS: https://api.signalstickers.com/feed/rss/
